lib/mp4/transmuxer.js

```diff
@@ -1114,7 +1111,7 @@ Transmuxer = function(options) {
     if (videoTrack) {
       if (pipeline.videoSegmentStream) {
         pipeline.videoSegmentStream.gopCache_ = [];
-        pipeline.videoTimestampRolloverStream.discontinuity();
+        pipeline.timestampRolloverStream.discontinuity();
```

Comment:
its probably safe to delete this line, since there is a check for the rolloverstream at the end of this function